examples/Cycling leg

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 28 19:36:25 2017
+
+@author: Anita Le Mair
+"""
+import numpy as np
+from scipy.integrate import odeint
+from sympy import sin, solve, symbols
+from sympy import Derivative, Matrix
+from sympy.physics.mechanics import (find_dynamicsymbols, dynamicsymbols,
+                                     inertia)
+from sympy.physics.mechanics import (Point, ReferenceFrame,
+                                     RigidBody, KanesMethod)
+from pydy.codegen.ode_function_generators import generate_ode_function
+
+# Symbolen
+theta, omega = dynamicsymbols('theta omega')
+q1, q2 = dynamicsymbols('q1 q2')
+u1, u2 = dynamicsymbols('u1 u2')
+r, m, g = symbols('r m g')
+bb,ob,ta,d1,d2 = symbols('bb ob ta d1 d2')
+t = symbols('t')
```

Comment:
it's better to pull t from `t = dynamicsymbols._t`

Reply:
done